schema.prisma

```diff
@@ -0,0 +1,87 @@
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+model User {
+  id                   Int           @id @default(autoincrement())
+  username             String        @unique
+  email                String?
+  hasPaid              Boolean       @default(false)
+  isUsingLn            Boolean       @default(false)
+  gptModel             String        @default("gpt-3.5-turbo")
+  datePaid             DateTime?
+  stripeId             String?
+  checkoutSessionId    String?
+  subscriptionStatus   String?
+  notifyPaymentExpires Boolean       @default(false)
+  credits              Int           @default(3)
+  letters              CoverLetter[]
+  jobs                 Job[]
+  lnData               LnData?
+  lnPayments           LnPayment[]
+}
+
+model LnData {
+  id        String    @id @default(uuid())
+  user      User?     @relation(fields: [userId], references: [id], onDelete: Cascade)
+  userId    Int?      @unique
+  loginUrl  String?   @unique
+  k1Hash    String    @unique
+  token     String?
+  createdAt DateTime  @default(now())
+  updatedAt DateTime? @updatedAt
+}
+
+// type LightningInvoice = {
+//   status: string;
+//   successAction: {
+//     tag: string;
+//     message: string;
+//   };
+//   verify: string;
+//   routes: any[]; // You can replace this with a more specific type if needed
+//   pr: string;
+// };
```

Comment:
@vincanger Didn't know where you wanted this so I kept the same location it had in the wasp file.